R/manual_layer_udf.R

```diff
@@ -47,6 +47,7 @@
 execute_generic_udf <- function(udf=NULL, registered_udf_name=NULL, args=NULL, result_format='native',
   args_format='native', namespace=NULL, language='r', resource_class=NULL)
 {
+  cat("EXECUTE_GENERIC_UDF RESOURCE_CLASS", resource_class, "\n")
```

Comment:
Maybe remove this / make it conditional on a debug flag?

Reply:
sorry this was debug stuff -- removed on a subsequent commit